src/base.iced

```diff
@@ -395,6 +401,7 @@ class Base
 
     err = @_v_check_user {json}
 
+    # TODO: I don't quite understand the flow here. Do I need to do checks for hprev here?
```

Comment:
In particular: is this a security check? Or input consistency check? Do I need to add stuff here?

Reply:
See `_check_sig` in `web/sig.iced`.  The caller fills in fields of what it expects the json should have.  And this function checks for correspondence.